packages/terraform-modules-public/gcp/testnet/modules/proxy/startup.sh

```diff
@@ -347,8 +347,8 @@ IN_MEMORY_DISCOVERY_TABLE_FLAG=""
 
 # Load configuration to files
 mkdir -p $DATA_DIR/account
-echo -n '${genesis_content_base64}' | base64 -d > $DATA_DIR/genesis.json
-echo -n '${bootnodes_base64}' | base64 -d > $DATA_DIR/bootnodes
+#echo -n '${genesis_content_base64}' | base64 -d > $DATA_DIR/genesis.json
```

Comment:
delete if not needed? or maybe a comment explaining why these comments & the others are still included if there's a reason

Reply:
Left these in here because handling of genesis block and bootnodes changed significantly in 1.1.0, and there's some ongoing disagreement about how to handle it going forward.  I clarified the reasoning in a comment.